### ISSUE TITLE
Get calendar event from user-specified app.

### ISSUE THREAD

**Issue by Crisis82** [8bitPit/Niagara-Issues#2116]
### Is your feature request related to a problem?

I'm using Simple Calendar Pro and the 'add event' feature from Niagara home widget make you choose which app you want to use to complete the action and everthing works correctly. On the other hand there isn't a prompt to make you choose from which app get the events data, in fact I have already added events in the calendar app, but none of them is showed and I think that's because Niagara doesn't know where to get the events from.

### Describe the solution you'd like

Would be useful to let the user decide from which calendar app gets the event to show in the home widget.

Another similar feature is to let the user decide which app to open when clicking on the extended widget info. For example, when clicking the weather info, would be nice to be redirected in the weather app that is choosed by the user.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by 8bitPit** [comment]
Hi @Crisis82, thanks for letting us know! We support all calendar apps that make use of Androids Calendar Provider API. This is a standardized way for all apps to access and modify calendar events on your device. Most, but unfortunately, make calendar events available through that API. I suggest contacting the developer of Simple Calendar Pro to support Android's Calendar API. Regarding your second request about our weather widget, we made this decision on purpose, see _Why is there no option to launch a weather app of my choice when tapping on Niagara's forecast widget?_:
: https://help.niagaralauncher.app/article/105-weather-widget#:~:text=Why%20is%20there%20no,Launcher%27s%20hourly%20weather%20forecast.

**Comment by Crisis82** [comment]
For the Calendar API I wrote to the developer and I'm waiting for an answer.
For the weather app I read the FAQ and now I have a more clear vision. For the app I know that if the user uses a different weather provider this will cause conflicts with the data, however, in my case i'm using OpenWeather as provider, i'm only not using their official app, but another FOSS project, which intent is to provide a clean and minimal interface. However I know that that's my personal preferences and I cannot pretend anything, but I tried asking :)
BTW, thanks a lot for making the app available also to non Google user, much appreciated!
